### PR TITLE
Fix single precision library

### PR DESCRIPTION
**Pull request by jschueller**
@jakirkham there was a problem with a missing macro in the config file

### CODE REVIEW

File: recipe/CMakeLists.txt [hunk 1]
```diff
@@ -66,7 +66,7 @@ check_prototype_definition (memalign "void *memalign(size_t alignment, size_t si
 check_prototype_definition (posix_memalign "int posix_memalign(void **memptr, size_t alignment, size_t size)" "0" stdlib.h HAVE_DECL_POSIX_MEMALIGN)
 
 include (CheckSymbolExists)
-check_symbol_exists (clock_gettime sys/time.h HAVE_CLOCK_GETTIME)
+check_symbol_exists (clock_gettime time.h HAVE_CLOCK_GETTIME)
 check_symbol_exists (gettimeofday sys/time.h HAVE_GETTIMEOFDAY)
```
Comment (jakirkham) [inline recipe/CMakeLists.txt]: Might need to do something [special]( https://stackoverflow.com/a/26085827 ) to get this to work on Windows. If this just for a random seed, `clock_gettime` should suffice.

Comment (jakirkham) [inline recipe/CMakeLists.txt]: Actually the one spot in the code already has a reasonable workaround for this case. So I think we are ok.

File: recipe/config.h.in [hunk 2]
```diff
@@ -91,10 +91,10 @@
 /* #undef HAVE_ARMV8CC */
 
 /* Define to enable AVX optimizations. */
-#cmakedefine01 HAVE_AVX
+#cmakedefine HAVE_AVX 1
 
 /* Define to enable AVX2 optimizations. */
-#cmakedefine01 HAVE_AVX2
+#cmakedefine HAVE_AVX2 1
```
Comment (jakirkham) [inline recipe/config.h.in]: These end up being the same thing either way, right?

**Comment by jakirkham** [comment]
Might want to check the `HAVE_CLOCK_GETTIME` definition in `config.h.in` too.

**Comment by jakirkham** [comment]
Also `FFTW_CC` is a bit strange.

File: recipe/config.h.in (hunk 2 above)
Comment (jschueller) [inline recipe/config.h.in]: when false, it still writes "#define HAVE_XXX 0"

**Comment by jakirkham** [comment]
Travis CI timed out. Not that it matters much as these changes don't affect that build. Have gone ahead and restarted it.

**Pull request merged**